public/editor/scripts/editor/js/fc/bramble-menus.js

```diff
@@ -78,7 +78,24 @@ define(function(require) {
 
       return false;
     });
+  $("#allow-lint-toggle").click(function() {
+    // Toggle current value
+     var $allowWSToggle = $("#allow-lint-toggle");
+     var toggle = !($allowWSToggle.hasClass("switch-enabled"));
+ 
+       if(toggle) {
```

Comment:
Indenting issues here too.  In Thimble we use 2-spaces. Your entire `if` block is indented 2-spaces too far.